src/Cli/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -122,7 +122,7 @@ private static Command GetTrustCommand()
             trustCommand.AddArgument(new Argument<string>() { Arity = ArgumentArity.ZeroOrOne }
                          .FromAmong(new string[] { "list", "author", "repository", "source", "certificate", "remove", "sync" }));
 
-            trustCommand.AddOption(new Option<string>("--algorithm"));
+            trustCommand.AddOption(new Option<string>("--algorithm").FromAmong("SHA256", "SHA384", "SHA512"));
```

Comment:
same here, the algorithms listed in nuget help are a thing that we can forward along.

it would be nice to consider asking the NuGet team for a System.Commandline Command that we could seamlessly integrate and not have to manually patch over like this.  @clairernovotny would this be a good thing to track?

Reply:
This issue is basically that: https://github.com/NuGet/Home/issues/10039

Problem from NuGet.Client's side is how to prioritize.

Reply:
Thank you for the link, @zivkan! Re: prioritization, I think it's a nice to have. If the NuGet team wants better/complete/up-to-date tab-suggestion that would be a good way to get it, but the current mechanism here seems to be mostly working.  Perhaps worth a maintenance task every so often to check the help output?